SingleRepo.java

```diff
@@ -0,0 +1,15 @@
+public synchronized static SingletonRepo getInstance() {
+    if (singletonrepo == null) {
+        singletonrepo = new SingletonRepo();
+
+        ac = new BasicAWSCredentials(ACCESS_KEY, SECERET_KEY);
+
+        dynamodb = AmazonDynamoDBClientBuilder
+                .standard()
+        .withRegion("us-west-2")
+        .withCredentials(new AWSStaticCredentialsProvider(ac) )
+        .build();
+        System.out.println("singleton repo instance created");
+    }
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;}&#34;, delete this token